projects/cds/README.md

```diff
@@ -27,3 +48,39 @@ To add a SAP Context-Driven Merchandising Carousel to a page following the steps
 * Fill in the fields of the configuration window that will be displayed, including selecting a configured strategy from the 'Strategy' drop down box.
 * Click the 'Save' button.
 * The rendered merchandising carousel should now be visible in the page you're editing.
+
+## Profiletag
```

Comment:
Can we mention that `PROFILE` consent template is expected to exist on the back-end, and otherwise the `CDS` [wouldn't work](https://sap-cx.slack.com/archives/DPZ7UAV54/p1573745915017900)?